bin/avro2kafka

```diff
@@ -10,8 +10,8 @@ option_parser = OptionParser.new do |opts|
   opts.banner = "Version #{Avro2Kafka::VERSION} of Avro2Kafka\n" \
     "Usage: #{File.basename(__FILE__)} [options] [file]"
 
-  opts.on('-b', '--broker BROKER', 'The Kafka broker, eg. localhost:9092. This value is required.') do |broker|
-    options[:broker] = broker
+  opts.on('-b', '--broker-list BROKER', 'The Kafka broker list. Format: host:port. This value is required.') do |broker_list|
+    options[:broker_list] = broker_list
```

Comment:
Well, maybe not. I take it back, `Avro2Kafka` is responsible for the CLI and it makes it easier to test.